src/python/pants/backend/docker/target_types.py

```diff
@@ -615,6 +618,73 @@ class DockerImageTarget(Target):
     )
 
 
+class DockerMirrorImagesSourcesField(MultipleSourcesField):
+    help = (
+        "Filename globs for Docker image sources text files, with one image name per line.\n\n"
```

Comment:
Why not just embed these directly in the option value? There is always the `@fromfile` convention if anyone wants to use a file.

Reply:
`@fromfile` is for config only?